bve/src/parse/train_dat/sections/device.rs

```diff
@@ -0,0 +1,94 @@
+use bve_derive::{FromKVPSection, FromKVPValueEnumNumbers};
+
+#[derive(Debug, Default, Clone, PartialEq, FromKVPSection)]
+pub struct DeviceSection {
+    #[kvp(bare)]
+    ats: AtsAvailable,
+    #[kvp(bare)]
+    atc: AtcAvailable,
+    #[kvp(bare)]
+    eb: EbAvailable,
+    #[kvp(bare)]
+    const_speed: ConstSpeedAvailable,
+    #[kvp(bare)]
+    hold_brake: HoldBrakeAvailable,
+    #[kvp(bare)]
+    re_adhesion_device: ReAdhesionDeviceType,
+    #[kvp(bare)]
+    load_compensating_device: String,
+    #[kvp(bare)]
+    pass_alarm: PassAlarmType,
+    #[kvp(bare)]
+    door_open_mode: DoorMode,
+    #[kvp(bare)]
+    door_close_mode: DoorMode,
+}
+
+#[derive(Debug, Clone, PartialEq, FromKVPValueEnumNumbers)]
+pub enum AtsAvailable {
```

Comment:
Ats?

This abbreviation might be obvious to BVE people, but it's not obvious to me. Could you either expand the abbreviation in the enum name, or add a comment explaining it?

Reply:
No idea what it stands for. I'd have to go poking around. It's some kind of automatic signalling used in japan.

Reply:
Added some docs